core/domain/question_fetchers.py

```diff
@@ -25,12 +25,22 @@
 from core.domain import state_domain
 from core.platform import models
 
+from typing import List, Optional, Tuple
+
+MYPY = False
+if MYPY: # pragma: no cover
+    from mypy_imports import question_models
+    from mypy_imports import skill_models
+
 (question_models, skill_models) = models.Registry.import_models(
     [models.NAMES.question, models.NAMES.skill])
 
 
 def get_questions_and_skill_descriptions_by_skill_ids(
-        question_count, skill_ids, offset):
+    question_count: int,
+    skill_ids: List[str],
+    offset: int
+) -> Tuple[List[Optional[question_domain.Question]], List[List[Optional[str]]]]:
```

Comment:
This seems as a bit complicated type, can it be moved into a type variable or simplified?

Reply:
Typevar is added!